### ISSUE TITLE
EAC3 Audio fails to transcode and plex stuck detecting intros

### ISSUE THREAD

**Issue by ItsRainingHP** [linuxserver/docker-plex#291]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior

Successfully scan and detect intros of EAC3 audio files.

## Current Behavior
When scanning media for intros PMS fails to scan any audio using EAC3. At first I thought this is because PMS still uses /tmp for EAC3 transcodes instead of /transcode folder. I have since mounted /tmp which seems to have fixed the issue temporarily but I still have errors when trying to detect intros of EAC3 files.

## Steps to Reproduce

1. Import any media containing EAC3 audio.
2. PMS will constantly be stuck scanning the file.
3. Check the logs for the EAC3 error.

## Environment
**OS:**
Unraid 6.9.2
**CPU architecture:** x86_64/arm32/arm64
x86 AMD CPU.
**How docker service was installed:**

Official docker from Community Apps page.

## Command used to create docker container (run/create/compose/screenshot)

![image](https://user-images.githubusercontent.com/79205731/144039835-9c0beb0e-e515-4be9-aaea-30931e9b82a5.png)
![image](https://user-images.githubusercontent.com/79205731/144041246-89a0a04c-ae98-4692-96ab-2761768c0c74.png)

Temp folder after mounting /tmp:
![image](https://user-images.githubusercontent.com/79205731/144040733-dd157875-d272-4d7d-b6f6-bc08f2fe9c75.png)
![image](https://user-images.githubusercontent.com/79205731/144040896-6658c659-0da0-485d-a132-6df45b528082.png)

## Docker logs

```Nov 30, 2021 00:20:27.999 [0x14b64c821b38] DEBUG - Activity: updated activity c86c14b0-3852-40c2-ace5-75cd1662af18 - completed 90.4% - Scanning Movies
Nov 30, 2021 00:20:28.000 [0x14b64c821b38] DEBUG - Scanner: Processing directory /movies/Bad Times at the El Royale (2018) (parent: yes)
Nov 30, 2021 00:20:28.000 [0x14b64c821b38] VERBOSE -     * Noting that 'Bad Times at the El Royale (2018)' was updated at 2021-10-06 16:51:12
Nov 30, 2021 00:20:28.000 [0x14b64c821b38] VERBOSE -     * Taking directory 'Bad Times at the El Royale (2018)' (2870) out of the map (228 left)
Nov 30, 2021 00:20:28.000 [0x14b652255b38] ERROR - [Transcoder] [eac3_eae @ 0x154219649b00] EAE timeout! EAE not running, or wrong folder? Could not read '/tmp/pms-0253ec2e-fe02-4d2f-ad8f-eab77e4d8c3a/EasyAudioEncoder/Convert to WAV (to 8ch or less)/1df457fb-e65d-4a16-b65e-a1ea7f1740c4_27015-0-2358.wav'
Nov 30, 2021 00:20:28.000 [0x14b64c821b38] VERBOSE - Adding file for scanner: "/movies/Bad Times at the El Royale (2018)/Bad Times at the El Royale (2018).en.srt"
Nov 30, 2021 00:20:28.000 [0x14b64c821b38] VERBOSE - Adding file for scanner: "/movies/Bad Times at the El Royale (2018)/Bad Times at the El Royale (2018).mp4"
Nov 30, 2021 00:20:28.000 [0x14b64c31ab38] ERROR - [Transcoder] [eac3_eae @ 0x154219649b00] error reading output
Nov 30, 2021 00:20:28.000 [0x14b64beb4b38] ERROR - [Transcoder] Error while decoding stream #0:1: I/O error```

**Comment by james-d-elliott** [comment]
This occurs due to the sysctl fs.inotify.max_user_watches being too low for the number of folders you have I believe.

**Comment by ItsRainingHP** [comment]
So I read that this could be the issue too. When I run the following on my plex directories I get the following:

![image](https://user-images.githubusercontent.com/79205731/144055039-8edb8c3a-aa0a-4f98-b6fc-3ba7afd10804.png)

I then got to the math portion which I likely did wrong. I have quite a lot of memory available so I would like to allocate enough for this not to be an issue. Based on forums I found the following command:

```
sudo sh
echo  fs.inotify.max_user_watches=262144  >> /etc/sysctl.conf
sysctl -p
```

To my understanding "In the above example, the table was configured for 262144 (256K) directories on a system with 8GB of main memory." I would like to provide the system with 32 GB of main memory space and 1024k directories. Can anyone help with the commands for this?

Thank you!
